features/share/access-controls/user-access-controls/README.md

```diff
@@ -0,0 +1,265 @@
+```
+
+The output of the command above will be lengthy, but will end with this message.
+
+```
+API error
+unauthorized access
+```
+
+As expected, Share's User Access Controls prevented the expanded capabilities. Why? Because your `default` policy does not include the `USER_SET_ROLE` action, even on your own user account.
+
+## Managing User Access Controls using Your Own GraphQL Client
+The Vendia Share CLI is a great utility that makes User, Uni, and Node management simple.  However, the Vendia Share control plane provides a GraphQL API with a rich set of operations, including for user management.  It's the same API the Share CLI uses, meaning anything you can do with the Share CLI you can do with any GraphQL client.
+
+You'll now repeat a subset of the activities from the CLI section, this time using a custom GraphQL client that runs on [Node.js](https://nodejs.org/).  While you'll only interact with a few Vendia Share control plane API operations, [the full set](https://www.vendia.net/docs/share/share-graphql-schema) is accessible using an identical approach.
+
+### Get Your Default Role
+
+This command will get information about your user's Vendia Share `default` role.
+```
+npm run getRole
+```
+
+You will see an output that looks similar to what you saw using the Share CLI.
+
+<details>
+<summary>Output</summary>
+
+```
+Successfully called Share
+
+UserId:  aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee
+Has role: default
+
+        Permitting: UNI_INVITE
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_QUERY
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_RESET
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_DELETE
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_GET
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_JOIN
+        On: UniResource(*.unis.vendia.net#*)
+
+        Permitting: UNI_CREATE
+        On: UniResource(*.unis.vendia.net#*)
+
+        Permitting: UNI_EVOLVE_SCHEMA
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_MUTATE
+        On: NameResource(you@*.domain.com)
+
+        Permitting: UNI_DELETE_NODE
+        On: NameResource(you@*.domain.com)
+
+        Permitting: USER_INVITE
+        On: NameResource(*@*.*.*)
+
+```
+</details>
+
+You can inspect the `getRole.js` files behind the `npm` command above to see the code examples in detail, including the use of a Vendia-authored library called [aws-signed-fetch](https://www.npmjs.com/package/@vendia/aws-signed-fetch/v/0.0.5).
+
+## Summary
+
+In this example you:
+
+* Learned more about Vendia Share User Access Controls
+* Used the Share CLI to interact with some of Vendia Share's user management operations, which are protected by User Access Controls 
+* Used a Node.js GraphQL client to interact with those same user management operations, demonstrating the ability to fully automate user management with any GraphQL-capable tool you choose
```

Comment:
just curious, "what is the reason for choosing js for scripting?" Usually, for admin tasks, python and shell are preferred. Are we observing that App Dev will be doing most of the Uni Management? Is that why?

Reply:
Just to mix in some more languages/platform examples throughout our `examples` repo.  We have an awful lot of python GraphQL client interactions already.  Node is very popular on the AppDev side so wanted to show some examples with that as well.